fakepkg

```diff
@@ -10,9 +10,10 @@ declare -r tmp_root='fakepkg'
 # Actual assembly function
 # $1: pkgname , $2: destination dir , $3 verbosity
 function fakebuild() {
+	pkgver=$(pacman -Qi --dbpath "${ROOT_DIR}/var/lib/pacman" ${package} 2>/dev/null | sed -n 2p | awk '{print $3}')
```

Comment:
Sorry, this is completely unrelated and originated from my initial first attempt, but the awk plus sed stuff can be written more neatly as `sed -n -e 's/Version\s*: \(.*\)/\1/p'`.

Reply:
Actually, I was thinking more at `awk 'NR == 2{print $3}'`. Not a big fan of escapes.

Reply:
Admittedly, the escapes are ugly. However, searching for `Version` instead of hard-coding line 2, is prettier in my opinion.

Reply:
Without a doubt. OTOH.. do we want to set `LC_MESSAGES` to `C.utf8` for this to work universally? Some users may use non-English locales even for `root` (not me :smile:)

Reply:
Very good point :see_no_evil: indeed there might be such users...
I guess `LANG=C` should be fine sufficient.